### DISCUSSION TITLE
Income Statement reports with scalability for many customers

### DISCUSSION THREAD

**Discussion by 3bears-data** [getredash/redash#6086] [General]
Hi guys, 
I have 3 doubts...

The first one is: with redash, can I build Profit and Loss Statement dashboards?

And second is: if I have more than 100 customers, each customer needs a P&L statements dashboard, what's the best way to deliver reports to customers so that each one only sees their dashboard? Do I need to upload an instance of Redash per client?

Third: Can I deploy an image locally in my machine just for test? I don't want to have fees with cloud machines when in development or testing the products.

**Comment by kvacc** [comment]
Hello @3bears-data, 
Did you find an answer for your 1st doubt?

**Comment by guidopetri** [comment]
1. Yes, you can build dashboards like that as long as you have access to the data
2. I don't know the answer to that because I don't know your infra :) so unfortunately I don't know.
3. Yes, redash has docker images you can launch with docker compose.
